config/routes.go

```diff
@@ -66,22 +66,23 @@ func setupAPI(e *echo.Echo) {
 	d.POST("/", controllers.CreateDatacenterHandler)
 	d.PUT("/:project", controllers.UpdateDatacenterHandler)
 	d.DELETE("/:project", controllers.DeleteDatacenterHandler)
-	d.DELETE("/:project/envs/:env", controllers.DeleteServiceHandler)
-	d.POST("/:project/envs/:env/reset/", controllers.ResetServiceHandler)
-	d.DELETE("/:project/envs/:env/force/", controllers.ForceServiceDeletionHandler)
-	d.PUT("/:project/envs/:env/", controllers.UpdateServiceHandler)
-	d.POST("/:project/envs/:env/sync/", controllers.SyncServiceHandler)
-	d.GET("/:project/envs/:env", controllers.GetServiceHandler)
-	d.GET("/:project/envs/:env/builds/", controllers.GetServiceBuildsHandler)
-	d.GET("/:project/envs/:env/builds/:build", controllers.GetServiceBuildHandler)
+	d.POST("/:project/envs/", controllers.CreateEnvHandler)
```

Comment:
It's actually slightly more complicated than is mentioned here. 
POST /projects/:project/envs/ is actually to create the project, but not a build. It's a seperate step to an apply and won't accept yaml.

To trigger a build, the endpoint should be /api/projects/:project/environments/:environment/builds/, 
which then creates/returns a build. 

This means we'll still have to process the yaml before applying if we want to keep the same simple cli interface.